menpo/io/test/io_import_test.py

```diff
@@ -302,6 +302,24 @@ def test_importing_PIL_1_no_normalize(is_file, mock_image):
 
 @patch('PIL.Image.open')
 @patch('menpo.io.input.base.Path.is_file')
+def test_importing_PIL_1_proper_conversion(is_file, mock_image):
+    from menpo.image import BooleanImage
+ 
+    arr = np.zeros((10, 10), dtype=np.uint8)
+    arr[4, 4] = 255
```

Comment:
Sorry if I'm being slow. The mocked loaded image is a `10x10` `False` image with a single `True` pixel at `[4, 4]`. So the loaded boolean image should be the same correct? Therefore, why is the `np.all` saying the entire image is `True`?